src/tekton/pipeline.ts

```diff
@@ -33,7 +46,7 @@ export class Pipeline extends TektonItem {
     const pipelineTrigger = data.map<Trigger>(value => ({
       name: value.metadata.name,
       resources: value.spec.resources,
-      params: value.spec.params ? value.spec.params : undefined,
+      parameters: value.spec.params ? value.spec.params : undefined,
```

Comment:
there you can use `value.spec.params ?? undefined` see [nullish-coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) But in this case is no sense to do that check.